index.html

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+      <meta charset="UTF-8">
+      <meta name="viewport" content="width=device-width, initial-scale=1.0">
+      <title>Project Day 2</title>
+        <link rel="stylesheet" href="style.css">
+    </head>
+    <body>
+
+    <header class="header">
+      <h3 class="logo">SJA</h3>
+      <nav class="nav">
+      <img src="img/Union.png" alt="Union"/>
+		</nav>
+	</header>
+
+
+  <section class="about">
+          <h1>Laura Jung</h1>
+          <h3>Software Developer</h3>
+          <p>I can help you build a product ,<br> feature or website Look through <br>some of my work and experience!<br> If you like what you see and have a <br>project you need coded, <br> don’t hestiate to contact me.
+          </p>
```

Comment:
The paragraph (e.g., "I'm a software ...") should have a p tag.
Please use colors and font-family for about section from the design page.